.prettierrc

```diff
@@ -2,7 +2,7 @@
   "bracketSpacing": true,
   "printWidth": 100,
   "singleQuote": true,
-  "tabWidth": 2,
```

Comment:
Why is tabWidth changed to 10?

Reply:
Was testing prettierrc but forgot to change it back.